File: Python/Stacks/list_stack.py
class Stack:
    def __init__(self):
        self.values = []

    def push(self, value):
        self.values.append(value)

    def pop(self):
        if self.values:
            return self.values.pop()
        return None

    def peek(self):
        if self.values:
            return self.values[-1]
        return None

    def size(self):
        return len(self.values)

File: JavaScript/Stacks/listStack.js
class ListStack{
    constructor(){
        this.values = []; 
    }

    push(data){
        this.values.push(data);
    }

    pop(){
        return this.values.pop()
    }

    printStack(){
        console.log(this.values);
    }

}

const stack = new ListStack();
stack.pop();
stack.push(1);
stack.push(2);
stack.push(3);
stack.push(4);
stack.printStack();
stack.pop();
stack.pop();
stack.printStack();
let x = stack.pop();
console.log(x);
stack.printStack();
